server/homebrew.api.js

```diff
@@ -55,28 +111,12 @@ const beforeNewSave = (account, brew)=>{
 	brew.text = mergeBrewText(brew);
 };
 
-const newLocalBrew = async (brew)=>{
-	const newHomebrew = new HomebrewModel(brew);
-	// Compress brew text to binary before saving
-	newHomebrew.textBin = zlib.deflateRawSync(newHomebrew.text);
-	// Delete the non-binary text field since it's not needed anymore
-	newHomebrew.text = undefined;
-
-	let saved = await newHomebrew.save()
-		.catch((err)=>{
-			console.error(err, err.toString(), err.stack);
-			throw `Error while creating new brew, ${err.toString()}`;
-		});
-
-	saved = saved.toObject();
-	saved.gDrive = false;
-	return saved;
-};
-
 const newGoogleBrew = async (account, brew, res)=>{
 	const oAuth2Client = GoogleActions.authCheck(account, res);
 
-	return await GoogleActions.newGoogleBrew(oAuth2Client, brew);
+	const newBrew = excludeGoogleProps(_.clone(brew));
```

Comment:
Why cloning here? Another symptom of Mongoose docs? Or are we worried about affecting the input brew? I don't see anything here where we need to preserve a separate original copy of the brew.

If we have a reason we need to preserve the input brew, perhaps the `_.clone` should occur inside `excludeGoogleProps` instead of having to pass in a clone each time.

Reply:
Good call, it should go inside the exclude method.